### ISSUE TITLE
Many wordings have never been translated because they're not in Transifex

### ISSUE THREAD

**Issue by Jerome-Herbinet** [nextcloud/groupfolders#3130]
Here are at least 3 old issues talking about untranslated wordings. 

They don't appear in Transifex despite a supposed (IMO, but I'm maybe mistaking) good syntax : https://github.com/nextcloud/groupfolders/issues/2131, https://github.com/nextcloud/groupfolders/issues/2372, https://github.com/nextcloud/groupfolders/issues/2166

I took an overview of the situation concerning untranslated wordings (seen in GF 18.0.0 / 17.0.2 / 16.0.7 ...) : 
- "Users/groups that can manage" (placeholder)
- "Unlimited" and "Other ..." (quota dropdown)
- To this, another translation isn't visible in Transifex  : 
![2024-08-21_16-05](https://github.com/user-attachments/assets/81bbbad2-6fbc-4fc9-9dd6-2496d6014bac)

### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by Jerome-Herbinet** [comment]
@icewind1991 ? @jancborchardt ?
This is not a critical problem, but it has been hampering the administration of group files for non-English speakers for years.

**Comment by jancborchardt** [comment]
Yes, should of course be translatable. FYI @icewind1991 @skjnldsv @sorbaugh

**Comment by smarinier** [comment]
@joshtrichards @jancborchardt this look (at least partially) like the same issue we just met with "bbb integration" ( https://github.com/littleredbutton/cloud_bbb/issues/261 ): indeed,  as the front of groupfolders is based on React, it uses backticks that are very badly handled by xgettext. When i run the "check-files" options from translationtool.phar, it describes the problems and the lines that is the source of the bug (that seem to feet with Jerome feedback):

```
** Warning : Check potentially missing translations sentences : groupfolders /var/www/html/nc29-dev/apps-writable/groupfolders/src/settings/App.tsx
> Starting at line 290
  -> t('groupfolders', 'Advanced permissions allows setting permissions on a per-file basis but comes with a performance overhead')
  -> t('groupfolders', 'Advanced permissions are only supported with Nextcloud 16 and up')
> Starting at line 298
  -> t('groupfolders', 'Delete')
  -> t('groupfolders', 'Folder name')
  -> t('groupfolders', 'Quota')
  -> t('groupfolders', 'Advanced Permissions')
  -> t('groupfolders', 'Folder name')
  -> t('groupfolders', 'Create')
  -> t('groupfolders', 'User')
  -> t('groupfolders', 'Group')
  -> t('groupfolders', 'Users/groups that can manage')
```

So i propose a very simple PR #3191 .

Note : maybe NextCloud should communicate to "React front apps" so that they should check their translations with translationtool ? (Do NextCloud know wich ones are using react ?)

**Comment by joshtrichards** [comment]
Thanks, @smarinier!

> Note : maybe NextCloud should communicate to "React front apps" so that they should check their translations with translationtool ? (Do NextCloud know wich ones are using react ?)

At least within the project itself [a quick and dirty search](https://github.com/search?q=org%3Anextcloud+%22from+%27react%27%22&type=code) shows only `groupfolders` and `whiteboard`.

**Comment by smarinier** [comment]
> At least within the project itself [a quick and dirty search](https://github.com/search?q=org%3Anextcloud+%22from+%27react%27%22&type=code) shows only `groupfolders` and `whiteboard`.

I just have a quick look. Whiteboard does not have l10n folder, neither seems to be linked to nextcloud transifex repository. And at least the translationtool doesn't seem to work on it. But it appears to be a very recent project (may 2024 ?)

**Comment by joshtrichards** [comment]
Good point. I was only looking for indications of React usage. :)
